### ISSUE TITLE
add lvm2 as explicit dependency

### ISSUE THREAD

**Issue by Jasper-Ben** [TheReal1604/disk-encryption-hetzner#16]
Depending on the image the hoster offers, lvm2 might not be installed by default. This caused me many hours of pain when I tried to adapt this guide for a different hoster. Unfortunately during debugging Debian / initramfs was not very helpful. Even though I explicitly added lvm2 to initramfs' modules I got no error message about this missing module while building the initramfs. Instead it just silently skipped on including the modules necessary for cryptsetup in initramfs causing `cryptroot-unlock` to fail with the cryptic error `timeout waiting for askpass.` You just can't imagine my frustration when it turned out that I spent hours debugging an error, which would have been fixed by a simple `apt install lvm2`.

TL;DR
don't assume that the hosters os images include lvm2 by default and explicitly add it in all guides: `apt install lvm2`

**Comment by TheReal1604** [comment]
Thanks for your contribution, I merge the PR now.
